File: public/docs/_examples/displaying-data/ts/app/app-ctor.component.ts
import {Component} from 'angular2/core';

@Component({
  selector: 'my-app-ctor',
  template: `
    <h1>{{title}} [Ctor version]</h1>
    <h2>My favorite hero is: {{myHero}}</h2>
    `
})
// #docregion app-ctor
export class AppCtorComponent {
  title: string;
  myHero: string;

  constructor() {
    this.title = 'Tour of Heroes';
    this.myHero = 'Windstorm';
  }
}
// #enddocregion app-ctor

File: public/docs/_examples/displaying-data/dart/lib/app_ctor_component.dart
// #docregion
library displaying_data.app_ctor_component;

import 'package:angular2/angular2.dart';

@Component(
    selector: 'my-app-ctor',
    template: '''
<h1>{{title}} [Ctor version]</h1>
<h2>My favorite hero is: {{myHero}}</h2>''')
// #docregion app-ctor
class AppCtorComponent {
  String title;
  String myHero;

  AppCtorComponent() {
    this.title = 'Tour of Heroes';
    this.myHero = 'Windstorm';
  }
}
// #enddocregion app-ctor
